python/ray/data/read_api.py

```diff
@@ -3249,6 +3250,77 @@ def read_lance(
     )
 
 
+@PublicAPI
```

Comment:
Let's annotate as  `@PublicAPI(stability="alpha")` to it to make it clear this isn't a stable API yet

Reply:
Done